airbyte-integrations/connectors/source-twilio/acceptance-test-config.yml

```diff
@@ -31,6 +31,8 @@ acceptance_tests:
             bypass_reason: "stream not filled yet"
           - name: "conference_participants"
             bypass_reason: "stream not filled yet"
+          - name: "conversation_participants"
+            bypass_reason: "stream not filled yet"
```

Comment:
please remove this. you're adding a new stream and we must validate it is working properly.